src/Libs/cairo-1.0/Records/SolidPattern.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Cairo
+{
+    public class SolidPattern : Pattern
+    {
+        private SolidPattern(Internal.PatternHandle handle)
+            : base(handle)
+        {
+        }
+
+        public static SolidPattern CreateRgb(double r, double g, double b)
```

Comment:
Why does this class has static create methods but other classes use a constructor?

Reply:
In the cairo docs there was a suggestion to do this:

> The functions that should map to constructors or static methods for the various types are listed after the type, methods on that type are listed below. Note that cairo_pattern_create_rgb() and cairo_pattern_create_rgba() should not be overloaded with each other as a SolidPattern() constructor, but should appear as static methods instead. This is to maintain code clarity by making it clear how the arguments relate to color components

But it does feel inconsistent so I'm not 100% sure if we should follow that.
FWIW, in GtkSharp these were done as overloaded constructors (along with further overloads for the `Cairo.Color` struct that they added for convenience)

Reply:
I think if it matches the official documentation it is okay 👍